app/services/user.js

```diff
@@ -1,17 +1,37 @@
 const bcrypt = require('bcryptjs');
 const User = require('../models').Users;
+const { validationError } = require('../errors');
+const { errorMessages } = require('../constants/errorMessages');
 
-const encryptPassword = async password => {
+exports.encryptPassword = async password => {
   const salt = await bcrypt.genSalt(10);
   return bcrypt.hash(password, salt);
 };
 
+const comparePassword = async (bodyPassword, databasePassword) => {
+  const compare = await bcrypt.compare(bodyPassword, databasePassword);
+  return compare;
+};
+
 exports.createUser = async data => {
-  const hash = await encryptPassword(data.password);
+  const hash = await this.encryptPassword(data.password);
   return User.create({
     firstName: data.first_name,
     lastName: data.last_name,
     email: data.email.trim().toLowerCase(),
     password: hash
   });
 };
+
+exports.createSession = async data => {
+  const user = await User.findOne({
+    where: {
+      email: data.email.trim().toLowerCase()
+    }
+  });
+
+  if (!user) throw validationError(errorMessages.userNotFound);
```

Comment:
Some people recommend not specifying this error, with an "invalid username and/or password", it would be enough